src/main/kotlin/com/terraformation/backend/customer/model/IndividualUser.kt

```diff
@@ -178,6 +179,14 @@ data class IndividualUser(
         (organizationId in organizationRoles)
   }
 
+  override fun canCreatePlantingSite(organizationId: OrganizationId): Boolean {
+    return when (organizationRoles[organizationId]) {
+      Role.OWNER,
+      Role.ADMIN -> true
+      else -> false
+    }
```

Comment:
optional suggestion: maybe extract this out .. since this pattern is used in many places?
```
fun hasElevatedPrivileges(orgId: OrganizationId): Boolean { .... }
```

Reply:
Agreed, this code would benefit from some refactoring (as would `PermissionRequirements`). A bit out of scope for this PR but we should do it.